### ISSUE TITLE
Rewriting

### ISSUE THREAD

**Issue by ThatNerdyPikachu** [ThatNerdyPikachu/nspbuild#15]
Hey all, I’ve decided to rewrite this tool entirely. It’s still going to be in Go (I need an excuse to start writing Go again, hah), and might have a GUI :eyes:

@neonsea Do you mind helping me with this, code review wise?

I also want the communities help on this, as I know tools already exist that are “better,” but I truly want to create something that is perfect.

Thanks everyone!

**Comment by ThatNerdyPikachu** [comment]
@SegFault42 you also had some good fixes that you PRd — could you help?

**Comment by SegFault42** [comment]
Hi,

Yes, can you create a project board like this :     
  https://github.com/Dontwait00/nXDownload/projects/2#column-4338924

I want to know what we have to do.

**Comment by ThatNerdyPikachu** [comment]
Yeah, I was going to do that once we had some ideas, this was more of a call for action, since I really have no clue what we should add support for. RetroArch forwarders maybe?

I want to make this the best tool, so, uhm..

Edit: If anyone reading has any ideas — make an issue with info about the feature and such, and I’ll move it to the board.

**Comment by nnsee** [comment]
Sure, I'll have a look.
